Automattic/jetpack
# Podcast Player block: Add `Align` (Wide & Full Width) and `Spacing` (Margin and Padding) tools

The proposed change adds the `Align` (Wide & Full Width) and `Spacing` (Margin & Padding) tools to the Podcast Player block:

 Before | After |
| ------------- | ------------- |
| ![Markup on 2022-02-03 at 15:52:56](https://user-images.githubusercontent.com/25105483/152366993-ef32ed09-c277-40c4-b9c1-7ec7a5a1f21c.png) | ![Markup on 2022-02-03 at 15:50:32](https://user-images.githubusercontent.com/25105483/152367042-6741ba02-4f98-4382-a227-c28bd7855d28.png) |
| ![Markup on 2022-02-09 at 14:51:24](https://user-images.githubusercontent.com/25105483/153214768-377f512f-608f-4238-ad25-00f36fe1d488.png) | ![Markup on 2022-02-09 at 14:50:41](https://user-images.githubusercontent.com/25105483/153214825-28a34608-9331-42a7-b62f-7adc9857bc07.png) |

#### Changes proposed in this Pull Request:
- add the `Align` (`Wide` & `Full Width`) and `Spacing` (Margin & Padding) `support`
- remove `alignWide: true` `support` that didn't work (and related comments)

#### Jetpack product discussion
This PR is a part of the _Expanding Jetpack Block design tooling for Full Site Editing_ project: pdDOJh-6-p2.

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Create a new post or a page with the Podcast Player block. You can use the following podcast feed URL: `https://distributed.blog/category/podcast/feed/`
2. The `Align` and `Spacing` (displayed as "Dimensions" in the block sidebar) tools should be available
3. Experiment with the tools and observe whether the change applies correctly in the back end
4. Save the changes and review them in the front end as well
5. We can test the change with multiple themes (block-based, classic). Please note that not all themes support the `Align` and `Spacing` tool. The `Geologist` theme supports both tools.
